components/post_view/channel_intro_message/add_members_button.tsx

```diff
@@ -89,6 +91,7 @@ const LessThanMaxFreeUsers = ({setHeader, pluginButtons}: {setHeader: React.Reac
                         className='intro-links color--link cursor--pointer'
                         modalId={ModalIdentifiers.INVITATION}
                         dialogType={InvitationModal}
+                        onClick={() => trackEvent('ui', 'ui_channel_intro_invite_button_clicked')}
```

Comment:
Is the category name too generic? Should it relate to the action?

Reply:
It is a `ui` element click though, like [many others](https://github.com/search?q=repo%3Amattermost%2Fmattermost-webapp%20trackEvent\(%27ui%27&type=code) we track. Do you have a suggestion for an alternative?

Reply:
I guess it is quite subjective because we don't really have any strict naming conventions. Personally whenever I'm adding telemetry I use the category to describe where we are in the webapp and the name describes the action. So for example in this case I would have put `trackEvent('channel_intro_message', 'invite_button_clicked')`. What's your opinion on that?

Reply:
Researched how we use these, and you're right. `ui` seems to be used mostly on the UI components that are always present. While in this case, it is part of some channels.